lib/src/query_component.dart

```diff
@@ -93,26 +100,20 @@ class QueryComponent {
       showAlert = false;
 
       // Uses DV360 public APIs to fetch entity data.
-      List<InsertionOrder> insertionOrders =
-          await _queryAndParseInsertionOrderEntityData();
-
-      // Early return if the query returns no insertion orders.
-      if (insertionOrders.isEmpty) return;
-
-      // If query type is not by insertion order, filters out insertion orders
-      // that are not in-flight.
-      insertionOrders = queryType == QueryType.byInsertionOrder
```

Comment:
This filter logic is moved to the [public api parser class](https://github.com/googleinterns/dv360-excel-plugin/pull/84/commits/cc16512f8afc9c6ea5415eef7df263df1d227762#diff-3863c7633dfe3db3e4718b1629ac87b5R55-R57).